fastlane/Fastfile

```diff
@@ -129,6 +126,13 @@ def release_version_next
   VERSION_FORMATTER.release_version(next_calculated_release_version)
 end
 
+def is_current_release_hotfix?
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/51531311b94a2f1dc631ed793f34a878423b96a5/fastlane/Fastfile#L129"></span>Naming/PredicateName: Rename <code>is_current_release_hotfix?</code> to <code>current_release_hotfix?</code>.</td>
    </tr>
  </tbody>
</table>